lib/galaxy/datatypes/tabular.py

```diff
@@ -691,6 +695,22 @@ def set_meta( self, dataset, **kwd ):
             # Found header line, get sample names.
             dataset.metadata.sample_names = line.split()[ 9: ]
 
+    @staticmethod
+    def merge(split_files, output_file):
+        tmp_dir = tempfile.mkdtemp()
```

Comment:
One final comment: does it make sense to create `tmp_dir` only to pass it to `NamedTemporaryFile()` ?